src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -561,6 +562,34 @@ def get_tank_ipv4(self, index: int) -> str:
         else:
             return None
 
+    def get_tank_dns_addr(self, index: int) -> str | None:
+        service_name = f"{self.network_name}-tank-{index:06d}-service"
```

Comment:
I originally went with "dns" because I figured "service" was k8s specific, and I wanted to keep the language generic and still accurate. That said, I'm totally fine renaming it.

Reply:
sorry I meant, there already is a get service name function you can use instead of building the string yourself